plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/RefreshVersionsCleanupTask.kt

```diff
@@ -53,7 +53,10 @@ open class RefreshVersionsCleanupTask : DefaultTask() {
 
     @TaskAction
     fun cleanUpVersionsCatalog() {
-        if (VersionsCatalogs.isSupported() && FeatureFlag.VERSIONS_CATALOG.isEnabled) {
+        val settingsFiles = OutputFile.settingsFiles
+            .filter { it.existed }
+
+        if (settingsFiles.contains(OutputFile.GRADLE_VERSIONS_CATALOG) && VersionsCatalogs.isSupported() && FeatureFlag.VERSIONS_CATALOG.isEnabled) {
```

Comment:
This doesn't look correct. The `OutputFIile.settingsFile` property contains an immutable list:
```kotlin
val settingsFiles = listOf(SETTINGS_GRADLE, SETTINGS_GRADLE_KTS, BUILD_SETTINGS_GRADLE, BUILD_SETTINGS_GRADLE_KTS)
```
So this expression you added will always bring `false`:
```kotlin
settingsFiles.contains(OutputFile.GRADLE_VERSIONS_CATALOG)
```

I know how to correctly fix #618 though, so I'll do that. Thank you for submitting this PR anyway, it put me in front of the right line of code, that's always saved time.

Reply:
@LouisCAD , Thanks for your kind reply.
I'm a little embarrassed that I made such a simple mistake, but thank you for telling me it was helpful nonetheless. 

If I find any issues while using it, I'll report back 🙂